smartsim/log.py

```diff
@@ -129,3 +129,23 @@ def log_to_file(filename: str, log_level: str = "debug") -> None:
         filename, "w+", encoding="utf-8"
     )
     coloredlogs.install(stream=stream, logger=logger, level=log_level)
+
+
+def log_to_file_preview(
```

Comment:
double-check that you still need this. i suggested it to avoid a collision with my changes that are incoming, but that was a bad choice on my part - i fixed my breaking changes to `log_to_file` - consider if you really need `log_to_file_preview`

Reply:
I think this is still needed. I did not remove

Reply:
removed